flake.nix

```diff
@@ -96,6 +96,16 @@
         '';
       };
 
+      haskell-flake = {
+        path = ./haskell-flake;
```

Comment:
This directory wasn't committed.

Reply:
I believe that directory was added in a prior PR. This PR corrects a mistake.